examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,135 @@
+# Basic Python CI With Cloud Source Repositories and Cloud Build
+
+## Overview
+This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
+
+The basic steps we need to follow are:
+1. Create a new project in Cloud Source Repository and clone it to your machine
+2. Create your code and unit tests
+3. Create a Python Cloud Builder (more below)
+4. Create a cloud build trigger (more below)
+5. Create a cloudbuild.yaml file that executes your tests using your python cloud builder.
+
+## 1. Create a new project in Cloud Source Repository
```

Comment:
Is there a way to do this with gcloud (and other steps)? If so it would be useful to have that information in here as well.

Reply:
Yes there is.  Added a gcloud example